src/main/java/uk/gov/register/configuration/ConfigManager.java

```diff
@@ -57,11 +52,11 @@ public FieldsConfiguration getFieldsConfiguration() {
         return fieldsConfiguration.get();
     }
 
-    private RegistersConfiguration createRegistersConfiguration() {
-        return new RegistersConfiguration(registersConfigFileUrl.map(s -> registersConfigFilePath));
+    private RegistersConfiguration createRegistersConfiguration() throws IOException {
```

Comment:
Should this throw `IOException` when the corresponding `createFieldsConfiguration` method doesn't?